misc/gui_linux.sh

```diff
@@ -0,0 +1,157 @@
+#!/bin/bash
+
+
+global=$(zenity --forms --add-entry="Round Name" --add-entry="Round Title" --add-entry="Main User" --add-entry="OS" --add-entry="Remote" --add-entry="Local" --add-entry="EndDate" --add-entry="NoDestroy" --separator=$'\034' --title=AeacusSE)
+
+IFS=$'\034' read -r name title user os remote local end destroy <<< "$global"
+
+next=""
+vulns=""
+
+while [[ "$next" != "None" ]]
+do
```

Comment:
Can we use `while [[]]; do` instead here?

Reply:
Could you elaborate?